Modules/readline.c

```diff
@@ -1568,6 +1571,11 @@ PyInit_readline(void)
         goto error;
     }
 
+    if (PyModule_AddStringConstant(m, "backend", backend) < 0)
+    {
+        goto error;
+    }
```

Comment:
```suggestion
    if (PyModule_AddStringConstant(m, "backend", backend) < 0) {
        goto error;
    }
```

pep7

Reply:
Ah, I was following the line above it. Sure that makes sense.